### ISSUE TITLE
Suggestion: Macro

### ISSUE THREAD

**Issue by AgarioIsBest1602** [AJS-development/Ogar-unlimited#1388]
I hate it i cant have macro for feed and split... can anybody add it?

**Comment by Agarian** [comment]
But we have one already... its all in `/src/settings/clientConfig.ini`

**Comment by AgarioIsBest1602** [comment]
Yes but thats u have to wait a second before it begins... i just want like i press Q i will double split

**Comment by Agarian** [comment]
Actually, the macro is the server...

**Comment by Megabyte918** [comment]
oh... well I searched for anything related to the "u have to wait a second before it begins" thing within the server and I couldn't find it, @AgarioIsBest1602  I would recommend just using a macro script for tampermonkey that allows for such macro's its much easier
